Dockerfile

```
@@ -1,205 +1,62 @@
+COPY . .
+
+# Compile the application
+RUN mix do compile, release
+
+# ----------------------------------------
+# 2. RUNTIME STAGE
+# ----------------------------------------
+FROM debian:bullseye-slim
+
+# Install runtime dependencies
+RUN apt-get update -y && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)